cmake/PyneMacros.cmake

```diff
@@ -201,7 +220,7 @@ macro(fast_compile _srcname _gnuflags _clangflags _otherflags)
     message(STATUS "Compiling ${_srcname} fast from assembly ${_asmname}")
     set(_filename "${_asmname}")
   endif()
-  set(PYNE_SRCS "${PYNE_SRCS}" "${_filename}")
+  set(PYNE_SRCS "${_filename}" "${PYNE_SRCS}")
```

Comment:
@gonuke @bam241, This line change is actually kinda of nice to have so that the ASM stuff tries to get built first. Also, in the event of a source compile, it ensures that the long compile starts first, rather than last, for max parallelism.